src/app/Components/ArtworkCard/ArtworkCard.tsx

```diff
@@ -226,13 +251,10 @@ export const ArtworkCard: React.FC<ArtworkCardProps> = memo(
       }, 300)
     }, [currentImageIndex])
 
-    if (!artwork || !artwork.images || artwork.images.length === 0) {
+    if (!artwork || !artwork.images || artwork.images.length === 0 || !displayImages) {
```

Comment:
```suggestion
    if (!artwork || !artwork.images || artwork.images.length === 0) {
```

I don't think !displayImages is needed here since `displayImages = artwork.images` and we check !artwork.images already